packages/storage/amplify_storage_s3/lib/src/S3PrefixResolver/amplify_storage_s3_prefix_resolver.dart

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'dart:async';
+
+import 'package:amplify_core/amplify_core.dart';
+
+abstract class AmplifyStorageS3PrefixResolver {
```

Comment:
Hui has updated the naming of this: https://github.com/aws-amplify/amplify-flutter/pull/2059/files#diff-778909e5ff89aa667611d1d080237284550304c40bb1d00667c9b42f37126dcc